src/contract.ts

```diff
@@ -45,35 +45,35 @@ export interface Artifact {
   /** version of artifact file */
   version: number;
   /** version of compiler used to produce this file */
-  compilerVersion: string;
+  compilerVersion?: string;
   /** build type, can be debug or release */
-  buildType: string;
+  buildType?: string;
   /** name of the contract */
   contract: string;
   /** md5 of the contract source code */
-  md5: string;
+  md5?: string;
```

Comment:
This is not recommended. If you don't need these fields, you can process the object.

Reply:
@freedomhero